src/platform/packages/private/kbn-grid-layout/grid/grid_row/grid_row.tsx

```diff
@@ -103,46 +102,43 @@ export const GridRow = React.memo(
          */
         const gridLayoutSubscription = gridLayoutStateManager.gridLayout$.subscribe(
           (gridLayout) => {
+            if (!gridLayout[rowIndex]) return;
             const newPanelIdsInOrder = getKeysInOrder(gridLayout[rowIndex].panels);
             if (panelIdsInOrder.join() !== newPanelIdsInOrder.join()) {
               setPanelIdsInOrder(newPanelIdsInOrder);
             }
           }
         );
 
-        const columnCountSubscription = gridLayoutStateManager.runtimeSettings$
```

Comment:
Moved this subscription to the other CSS variables subscription in `useGridLayoutState`